Kudu.Services/Docker/DockerController.cs

```diff
@@ -25,7 +26,7 @@ public DockerController(ITraceFactory traceFactory, IDeploymentSettingsManager s
         }
 
         [HttpPost]
-        public HttpResponseMessage ReceiveHook()
+        public async Task<HttpResponseMessage> ReceiveHook()
```

Comment:
>ReceiveHook [](start = 47, length = 11)

@mathewc@microsoft.com : The frontend intercepts /docker/hook requests. Do you see this being called?

Reply:
I will be doing e2e tests before committing, but my understanding from talking with Jenny is that the front end intercepts but also forwards.

Reply:
@sanchitmehta can confirm this. But my understanding is that the intercepted requests will not be forwarded to the worker. Also, I believe all this code (DockerController.cs) will go away soon as the new implementation on the front-end will be the default behavior going forward.

Reply:
I was unaware of that. I'll sync up with you guys offline - if this is truly an obsolete hook implementation, then I'll need to do this in the front end rather than Kudu.